src/components/Books.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Books = () => (
+  <h1>Books Page</h1>
+);
+
+export default Books;
```

Comment:
- [ ] It looks like we can work more on book.js: 🐱 

- Takes the book data as a prop and renders the title and the author
- Has a button for removing books

`the button wouldn't remove the books yet, this is just a placeholder for now`